themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -471,6 +471,10 @@ use-jwks-url=Use JWKS URL
 use-jwks-url.tooltip=If the switch is on, client public keys will be downloaded from given JWKS URL. This allows great flexibility because new keys will be always re-downloaded again when client generates new keypair. If the switch is off, public key (or certificate) from the Keycloak DB is used, so when client keypair changes, you always need to import new key (or certificate) to the Keycloak DB as well.
 jwks-url=JWKS URL
 jwks-url.tooltip=URL where client keys in JWK format are stored. See JWK specification for more details. If you use Keycloak client adapter with "jwt" credential, you can use URL of your app with '/k_jwks' suffix. For example 'http://www.myhost.com/myapp/k_jwks' .
+use-jwks=Use JWKS
+use-jwks.tooltip=If the switch is on, client public keys will be configurable in JWKS.
```

Comment:
Please add note to this tooltip, that switch "Use JWKS" is mutually exclusive with the switch "Use JWKS URL". Also please update the tooltip for switch "Use JWKS URL" in same manner.

Reply:
I added.